src/chat.rs

```diff
@@ -3395,25 +3400,28 @@ pub async fn get_past_chat_contacts(context: &Context, chat_id: ChatId) -> Resul
     Ok(list)
 }
 
-/// Creates a group chat with a given `name`.
+/// Creates an encrypted group chat.
 pub async fn create_group_chat(context: &Context, name: &str) -> Result<ChatId> {
-    let is_encrypted = true;
-    create_group_ex(context, is_encrypted, name).await
+    let encryption = Some(create_id());
+    create_group_ex(context, Sync, encryption, name).await
 }
 
-/// Creates a new unencrypted group chat.
+/// Creates an unencrypted group chat.
 pub async fn create_group_chat_unencrypted(context: &Context, name: &str) -> Result<ChatId> {
-    let is_encrypted = false;
-    create_group_ex(context, is_encrypted, name).await
+    let encryption = None;
+    create_group_ex(context, Sync, encryption, name).await
 }
 
 /// Creates a group chat.
 ///
-/// * `is_encrypted` - If true, the chat is encrypted (with key-contacts).
+/// * `sync` - Whether a multi-device synchronization message should be sent. Ignored for
+///   unencrypted chats currently.
+/// * `encryption` - If `Some(grpid)`, the chat is encrypted (with key-contacts).
 /// * `name` - Chat name.
 pub(crate) async fn create_group_ex(
     context: &Context,
-    is_encrypted: bool,
+    sync: sync::Sync,
+    encryption: Option<String>,
```

Comment:
Can be `grpid: String` directly with a documentation comment explaining that empty string means creating unencrypted group. It is anyway immediately converted into a `String` with `let grpid = encryption.unwrap_or_default();` at the beginning of the function.